scripts/check_test_inclusion.py

```diff
@@ -0,0 +1,89 @@
+#!/bin/python
+
+"""
```

Comment:
nit: The summary fragment typically appears on the same line as the triple quotes, i.e.:
```
"""Verifies ... file."""
```

Reply:
Done. I checked this out on a gLinux host and ran the whole file through `gpylint` too.